doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+****************************************
+NumPy: The Absolute Basics for Beginners
+****************************************
+
+Welcome to the absolute beginner's guide to NumPy! If you have comments or suggestions, please don’t hesitate to reach out!
+
+-----
+
+.. image:: images/NumPy_logo.svg
+
+-----
+
+
+Welcome to NumPy!
+-----------------
+
+NumPy (**Numerical Python**) is an open source Python library that's used in almost every field of science and engineering. It's the universal standard for working with numerical data in Python, and it's at the core of the scientific Python and PyData ecosystems. NumPy users include everyone from beginning coders to experienced researchers doing state-of-the-art scientific and industrial research and development. The NumPy API is used extensively in Pandas, SciPy, Matplotlib, scikit-learn, scikit-image and most other data science and scientific Python packages. 
+
+The NumPy library contains multidimensional array and matrix data structures (you'll find more information about this in later sections). It provides **ndarray**, a homogeneous n-dimensional array object, with methods to efficiently operate on it. NumPy can be used to perform a wide variety of mathematical operations on arrays.  It adds powerful data structures to Python that guarantee efficient calculations with arrays and matrices and it supplies an enormous library of high-level mathematical functions that operate on these arrays and matrices. 
+
+`Learn more about NumPy here <https://docs.scipy.org/doc/numpy-1.17.0/user/whatisnumpy.html>`_!
+
+Installing NumPy
+----------------
+  
+To install NumPy, we strongly recommend using a scientific Python distribution. If you're looking for the full instructions for installing NumPy on your operating system, you can `find all of the details here <https://www.scipy.org/install.html>`_.
+
+
+  
+If you already have Python, you can install NumPy with
+
+::
+
+  conda install numpy
+  
+or 
+
+::
+
+  pip install numpy
+  
+If you don't have Python yet, you might want to consider using `Anaconda <https://www.anaconda.com/>`_. It's the easiest way to get started. The good thing about getting this distribution is the fact that you don’t need to worry too much about separately installing NumPy or any of the major packages that you’ll be using for your data analyses, like pandas, Scikit-Learn, etc.
+
+You can find all of the installation details in the `Installation <https://www.scipy.org/install.html>`_ section at scipy.org.
```

Comment:
might as well make the `scipy.org` at the end of the sentence a link too

Reply:
Got it!